client/src/components/Battlefield.js

```diff
@@ -6,59 +6,172 @@ import Card from './Card.js';
 
 import { Zones } from '../helpers';
 
+import '../styles/Battlefield.css';
+
 class Battlefield extends Component {
   constructor(props) {
     super(props);
 
-    const { cards } = this.props;
+    this.PlayerInfoBox = this.PlayerInfoBox.bind(this);
+
+    const { cards, player } = this.props;
     this.state = {
       cards,
+      player,
     };
   }
 
   componentWillReceiveProps(nextProps) {
-    const { cards } = nextProps;
+    const { cards, player } = nextProps;
     this.setState({
       cards,
+      player,
     });
   }
 
+  PlayerInfoBox(props) {
+    const { player, shouldRender, position } = props;
+
+    let positionClass = 'leftInfoBar';
+    if (position === 'right') {
+      positionClass = 'rightInfoBar';
+    }
+
+    if (shouldRender && player) {
```

Comment:
nit: I'd format this as a [conditional render](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator).